scripts/updateUsers.ts

```
@@ -1,15 +1,14 @@
 import 'dotenv/config';
 import 'reflect-metadata';
-import * as fs from 'fs';
 import { getDb } from '../lib/db';
 import { User } from '../lib/entities/User';
 import { UserStats } from '../lib/entities/UserStats';
 import { Worker } from '../lib/entities/Worker';
 import { WorkerStats } from '../lib/entities/WorkerStats';
 import { convertHashrate } from '../utils/helpers';
+import { CKPoolAPI, CKPoolError, CKPoolErrorCode } from '../lib/ckpool';
```

ESLint

[error] `../lib/ckpool` import should occur before import of `../lib/db`

(import/order)